site/en/adminGuide/monitor/metrics_dashboard.md

```diff
@@ -0,0 +1,163 @@
+  
+</details>
+
+<details><summary><h2>Query coordinator</h2></summary>
+  
+  | Panel  | Panel description  | PromQL (Prometheus query lanage)  | The Milvus metrics used  | Milvus metrics description  |
+|---|---|---|---|---|
+| Collection Loaded Num  | The number of collections that are currently loaded into memory.    | ```  sum(milvus_querycoord_collection_num{app_kubernetes_io_instance=~"$instance", app_kubernetes_io_name="$app_name", namespace="$namespace"}) by (app_kubernetes_io_instance)  ```  | `milvus_querycoord_collection_num`  |  The number of collections that are currently loaded by Milvus.    |
+| Entity Loaded Num  | The number of entities that are currently loaded into memory.    | ```  sum(milvus_querycoord_entitiy_num{app_kubernetes_io_instance=~"$instance", app_kubernetes_io_name="$app_name", namespace="$namespace"}) by (app_kubernetes_io_instance)  ```  | `milvus_querycoord_entitiy_num`  |  The number of entities that are currently loaded by Milvus.    |
+| Load Request Rate  | The number of load requests per minute within the past two minutes.    | ```  sum(increase(milvus_querycoord_load_req_count{app_kubernetes_io_instance=~"$instance", app_kubernetes_io_name="$app_name", namespace="$namespace"}[2m])/2) by (status)  ```  | `milvus_querycoord_load_req_count`  | The accumulated number of load requests.    |
+| Release Request Rate  | The number of release requests per minute within the past two minutes.    | ```  sum(increase(milvus_querycoord_release_req_count{app_kubernetes_io_instance=~"$instance", app_kubernetes_io_name="$app_name", namespace="$namespace"}[2m])/2) by (status)  ```  | `milvus_querycoord_release_req_count`  | The accumulated number of release requests.    |
+| Load Request Latency  | The 99th percentile of load request latency within the past two minutes.    | ```  histogram_quantile(0.99, sum by (le) (rate(milvus_querycoord_load_latency_bucket{app_kubernetes_io_instance=~"$instance", app_kubernetes_io_name="$app_name", namespace="$namespace"}[2m])))  ```  | `milvus_querycoord_load_latency`  | The time used to complete a load request.    |
+| Release Request Latency  | The 99th percentile of release request latency within the past two minutes.    | ```  histogram_quantile(0.99, sum by (le) (rate(milvus_querycoord_release_latency_bucket{app_kubernetes_io_instance=~"$instance", app_kubernetes_io_name="$app_name", namespace="$namespace"}[2m])))  ```  | `milvus_querycoord_release_latency`  | The time used to complete a release request.    |
+| Sub-Load Task  | The number of sub load tasks.    | ```  sum(milvus_querycoord_child_task_num{app_kubernetes_io_instance=~"$instance", app_kubernetes_io_name="$app_name", namespace="$namespace"}) by (app_kubernetes_io_instance)  ```  | `milvus_querycoord_child_task_num`  | The number of sub load tasks.    <br/>    A query coord splits a load request into multiple sub load tasks.   |
```

Comment:
Is the hyphen needed or not in the word "Sub-load"?

Reply:
This is the panel name as displayed in Prometheus and Grafana.